### PR TITLE
I added google fonts, custom theme, transitions, padding on header and my name on contributors file.

### PR DESCRIPTION
**Pull request by sauravk7077**

### CODE REVIEW

**Comment by terrifricker** [comment]
Thank you for contributing!  I'll have a look and get back to you today.

File: home-page/style.css [hunk 1]
```diff
@@ -38,6 +39,7 @@ body::-webkit-scrollbar-thumb {
 .header {
   background-color: #900e78;
   color: white;
+  padding-top: 10px;
```
Comment (terrifricker) [inline home-page/style.css:42]: This improves the look a lot.  Thanks for catching it.

**Review by terrifricker — COMMENTED** [review]
Saurav, I really like your theme and that you customized even more than any of us had in the past and added the new attributes to the jquery.

**Comment by terrifricker** [comment]
I am willing to merge your pull request, but _could_ ask for one change.  You could change the 2px rounded corners back into 5px in the main page, then make the 2px corners only in your theme.  But it is a small enough change that I could merge your changes as they are.

Let me know.

**Comment by DavidMatthewFraser** [comment]
Really great changes! The Abyss theme looks awesome! I honestly like the 2px border-radius, I just think that we should be consistent with button style (change it on one page, change it on all the pages).

File: home-page/style.css (hunk 1 above)
Comment (sauravk7077) [reply home-page/style.css:42]: Welcome

**Comment by sauravk7077** [comment]
I think 2px border-radius looks much better. I'll create a new field for border-radius after you much this.👍👍

**Pull request merged**